### PR TITLE
Update cloud database service

### PR DESCRIPTION
**Pull request by trisduong**

### CODE REVIEW

File: cloud_database.go [hunk 1]
```diff
@@ -0,0 +1,1282 @@
...
+}
+
+type CloudDatabaseService interface {
+	Instances() *cloudDatabaseInstances
+	Nodes() *cloudDatabaseNodes
+	Tasks() *cloudDatabaseTasks
+	Configurations() *cloudDatabaseConfigurations
+	Backups() *cloudDatabaseBackups
+	Schedules() *cloudDatabaseSchedules
+	Autoscalings() *cloudDatabaseAutoscalings
```
Comment (BoTranVan) [inline cloud_database.go:38]: Autoscaling -> AutoScaling

File: cloud_database.go [hunk 2]
```diff
@@ -0,0 +1,1282 @@
...
+
+func (db *cloudDatabaseService) Backups() *cloudDatabaseBackups {
+	return &cloudDatabaseBackups{client: db.client}
+}
+
+func (db *cloudDatabaseService) Schedules() *cloudDatabaseSchedules {
+	return &cloudDatabaseSchedules{client: db.client}
+}
+
+func (db *cloudDatabaseService) Autoscalings() *cloudDatabaseAutoscalings {
```
Comment (BoTranVan) [inline cloud_database.go:67]: ditto

File: cloud_database.go [hunk 3]
```diff
@@ -0,0 +1,1282 @@
...
+	Detailed        bool   `json:"detailed,omitempty"`
+	EndTime         string `json:"end_time,omitempty"`
+	Name            string `json:"name,omitempty"`
+	Page            int    `json:"page,omitempty"`
+	ResultsPerPages int    `json:"results_per_pages,omitempty"`
+	StartTime       string `json:"start_time,omitempty"`
+}
+
+// CloudDatabaseAutoscalingVolume contains volume information of autoscaling.
+type CloudDatabaseAutoscalingVolume struct {
```
Comment (BoTranVan) [inline cloud_database.go:198]: ditto

File: cloud_database.go [hunk 4]
```diff
@@ -0,0 +1,1282 @@
...
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+	var nodes *CloudDatabaseNode
+	if err := json.NewDecoder(resp.Body).Decode(&nodes); err != nil {
+		return nil, err
+	}
+	return nodes, nil
+}
```
Comment (BoTranVan) [inline cloud_database.go:679]: Need validate payload before sent it to API server

File: cloud_database.go [hunk 5]
```diff
@@ -0,0 +1,1282 @@
+// This file is part of goBizFly
```
Comment (BoTranVan) [inline cloud_database.go:1]: All func need validate payload before sent it to API server

**Review by BoTranVan — COMMENTED** [review]
@trisduong

File: cloud_database.go (hunk 4 above)
Comment (trisduong) [reply cloud_database.go:679]: I don't understand this. is all Payload already validated with struct when user use it?

**Review by BoTranVan — APPROVED** [review]
LGTM

**Pull request merged**